ci_js/doc/index.rst

```diff
@@ -0,0 +1,60 @@
+========
+ CI JS
+========
+
+
+Configuration
+=============
+
+
+Usage
+=====
+
+Создание нового тура
+--------------------
+
+* Открыть в верхнем меню ``[[ CI JS ]]``
+* В боковом меню во вкладке ``CI JS`` выбрать кнопку ``Tours``
+* Нажать на кнопку ``Create``
+* В открывшемся окне заполнить поля:
```

Comment:
All fields (with descriptions) you can find in https://github.com/odoo/odoo/blob/11.0/addons/web_tour/static/src/js/tour_manager.js#L173